components/automate-es-gateway/habitat/plan.sh

```diff
@@ -11,10 +11,10 @@ pkg_license=("Chef-MLSA")
 pkg_upstream_url="https://www.chef.io/automate"
 
 
-jdomain_version="1.1.5"
-jdomain_filename="ngx_upstream_jdomain-${jdomain_version}.tar.gz"
-jdomain_source="https://github.com/nicholaschiasson/ngx_upstream_jdomain/archive/${jdomain_version}.tar.gz"
-jdomain_shasum=e7346dac41d473ea02995b7626bd0660115538c2c784dfe683e235c58dbc2c2c
+jdomain_version="master"
```

Comment:
I didn't find any tag/version associated in the Repo to refer so going along with master.